runtime/dolphin/src/weights/pallet_utility.rs

```diff
@@ -88,4 +94,9 @@ impl WeightInfo for () {
 	fn dispatch_as() -> Weight {
 		(10_828_000 as Weight)
 	}
+	fn force_batch(c: u32, ) -> Weight {
```

Comment:
@grenade @Dengjianping seems there's a new extrinsic in pallet-utility to force execute all calls in a batch even if some of them fail. Useful for the collators batches.